### PR TITLE
Luminor payment logo

### PR DESCRIPTION
**Pull request by martabelfm**
** Checklist **
- [x] All icons have a corresponding entry in `db/payment_icons.yml`
- [x] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [x] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [x] I am confident that all icons are clear and easy to read/understand
- [x] I have provided a link to the brand icon’s brand guidelines whenever possible.

Luminor sent me this logo

If this pull request is not adding new icons, you can remove this checklist.

### CODE REVIEW

**Comment by tauthomas01** [comment]
Hi @martabelfm ,

Please follow the guideline (https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md), it's missing:

- Background
- Border around the icon.

**Comment by martabelfm** [comment]
> Hi @martabelfm ,
> 
> Please follow the guideline (https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md), it's missing:
> 
> * Background
> * Border around the icon.

I added border and background. Hopefully got it right this time.

**Comment by martabelfm** [comment]
@tausthomas01 

> Hi @martabelfm ,
> 
> Please follow the guideline (https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md), it's missing:
> 
> * Background
> * Border around the icon.

Hey, could you take another look if it's good now?

File: app/assets/images/payment_icons/luminor.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" role="img" viewBox="0 0 38 24" width="38" height="24" aria-labelledby="pi-luminor"><title id="pi-luminor">Luminor</title>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/luminor.svg:1]: Is it possible to add more spacing on the left and right side? It looks cramped

[alt](https://screenshot.click/02-00-jqc7b-zhlwr.png)

Also, you would need to run the svg code through SVGO like in the [contribution](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md#contributing) document.

Comment (martabelfm) [reply app/assets/images/payment_icons/luminor.svg]: I ran the code through SVGO and added more spacing. As the SVG code is not very known for me I used https://vectorpaint.yaks.co.nz/ to add spacing. Hopefully it's good, please let me know :)

File: app/assets/images/payment_icons/luminor.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...0-1.501.632v.05a1.612 1.467 0 00-1.51-.64 1.584 1.442 0 00-1.492.632v.05l-.389-.573h-.806v4.217h1.177v-2.15c0-.844.306-1.19.926-1.19s.843.32.843.987v2.269h1.288v-2.092c0-.793.296-1.155.926-1.155.371 0 .871.101.871.986v2.26h1.26V12.69c0-1.164-.5-1.686-1.593-1.686zm-7.82 2.184a1.316 1.198 0 01-.472 1.062 1.26 1.147 0 01-.583.118c-.732 0-.927-.38-.927-.995v-2.251H7.18v2.589c0 1.163.639 1.686 1.77 1.686a1.52 1.383 0 001.51-.683l.388.574h.806v-4.166h-1.25zm19.456-2.184a2.52 2.294 0 00-1.853.649 2.437 2.218 0 00-.658 1.552 2.4 2.184 0 00.658 1.543 2.52 2.294 0 001.853.65 2.353 2.142 0 002.501-2.193 2.353 2.142 0 00-2.501-2.201zm0 3.423a1.25 1.138 0 01-1.232-1.222 1.25 1.138 0 112.492 0 1.26 1.147 0 01-1.242 1.222zM3.566 9.435h-1.26v5.853h4.437v-1.19H3.566zm16.888 5.853h1.307V11.07h-1.307zm13.628-3.601l-.38-.574h-.824v4.217h1.26v-1.62a1.529 1.391 0 01.472-1.172 1.76 1.602 0 011.084-.362v-1.172c-1.352 0-1.621.683-1.621.683zM20.186 9.435l.926.843.927-.843-.927-.843zm5.003 1.577a1.538 1.4 0 00-1.51.675l-.39-.565h-.806v4.166h1.26V13.22a1.297 1.18 0 01.473-1.062 1.13 1.029 0 01.583-.127c.723 0 .927.38.927.995v2.26h1.223V12.69c0-1.164-.63-1.678-1.77-1.678z" fill="#481335" id="svg_3"/></g></svg>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/luminor.svg:1]: Can you remove the unused `id` attributes and add a white background?

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" role="img" width="38" height="24" aria-labelledby="pi-luminor"><rect width="100%" height="100%" x="0" y="0" fill="none" stroke="none"/><title id="pi-luminor">Luminor</title><g class="currentLayer"><path d="M35 0H3C1.3 0 0 1.3 0 3v18c0 1.7 1.4 3 3 3h32c1.7 0 3-1.3 3-3V3c0-1.7-1.4-3-3-3z" opacity=".07"/><path fill="#fff" d="M35 1c1.1 0 2 .9 2 2v18c0 1.1-.9 2-2 2H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h32" fill-opacity=".1" stroke-opacity=".9"/><path d="M18.222 11.004a1.593 1.45 0 00-1.501.632v.05a1.612 1.467 0 00-1.51-.64 1.584 1.442 0 00-1.492.632v.05l-.389-.573h-.806v4.217h1.177v-2.15c0-.844.306-1.19.926-1.19s.843.32.843.987v2.269h1.288v-2.092c0-.793.296-1.155.926-1.155.371 0 .871.101.871.986v2.26h1.26V12.69c0-1.164-.5-1.686-1.593-1.686zm-7.82 2.184a1.316 1.198 0 01-.472 1.062 1.26 1.147 0 01-.583.118c-.732 0-.927-.38-.927-.995v-2.251H7.18v2.589c0 1.163.639 1.686 1.77 1.686a1.52 1.383 0 001.51-.683l.388.574h.806v-4.166h-1.25zm19.456-2.184a2.52 2.294 0 00-1.853.649 2.437 2.218 0 00-.658 1.552 2.4 2.184 0 00.658 1.543 2.52 2.294 0 001.853.65 2.353 2.142 0 002.501-2.193 2.353 2.142 0 00-2.501-2.201zm0 3.423a1.25 1.138 0 01-1.232-1.222 1.25 1.138 0 112.492 0 1.26 1.147 0 01-1.242 1.222zM3.566 9.435h-1.26v5.853h4.437v-1.19H3.566zm16.888 5.853h1.307V11.07h-1.307zm13.628-3.601l-.38-.574h-.824v4.217h1.26v-1.62a1.529 1.391 0 01.472-1.172 1.76 1.602 0 011.084-.362v-1.172c-1.352 0-1.621.683-1.621.683zM20.186 9.435l.926.843.927-.843-.927-.843zm5.003 1.577a1.538 1.4 0 00-1.51.675l-.39-.565h-.806v4.166h1.26V13.22a1.297 1.18 0 01.473-1.062 1.13 1.029 0 01.583-.127c.723 0 .927.38.927.995v2.26h1.223V12.69c0-1.164-.63-1.678-1.77-1.678z" fill="#481335"/></g></svg>
```

**Comment by tauthomas01** [comment]
Can we add a white background to follow like other icons? https://screenshot.click/28-21-xbt6m-be65e.mp4

**Comment by martabelfm** [comment]
> Can we add a white background to follow like other icons? https://screenshot.click/28-21-xbt6m-be65e.mp4

Should be okay now. Thanks for the feedback.

File: app/assets/images/payment_icons/luminor.svg [hunk 3]
```diff
@@ -0,0 +1 @@
...93 1.45 0 00-1.501.632v.05a1.612 1.467 0 00-1.51-.64 1.584 1.442 0 00-1.492.632v.05l-.389-.573h-.806v4.217h1.177v-2.15c0-.844.306-1.19.926-1.19s.843.32.843.987v2.269h1.288v-2.092c0-.793.296-1.155.926-1.155.371 0 .871.101.871.986v2.26h1.26V12.69c0-1.164-.5-1.686-1.593-1.686zm-7.82 2.184a1.316 1.198 0 01-.472 1.062 1.26 1.147 0 01-.583.118c-.732 0-.927-.38-.927-.995v-2.251H7.18v2.589c0 1.163.639 1.686 1.77 1.686a1.52 1.383 0 001.51-.683l.388.574h.806v-4.166h-1.25zm19.456-2.184a2.52 2.294 0 00-1.853.649 2.437 2.218 0 00-.658 1.552 2.4 2.184 0 00.658 1.543 2.52 2.294 0 001.853.65 2.353 2.142 0 002.501-2.193 2.353 2.142 0 00-2.501-2.201zm0 3.423a1.25 1.138 0 01-1.232-1.222 1.25 1.138 0 112.492 0 1.26 1.147 0 01-1.242 1.222zM3.566 9.435h-1.26v5.853h4.437v-1.19H3.566zm16.888 5.853h1.307V11.07h-1.307zm13.628-3.601l-.38-.574h-.824v4.217h1.26v-1.62a1.529 1.391 0 01.472-1.172 1.76 1.602 0 011.084-.362v-1.172c-1.352 0-1.621.683-1.621.683zM20.186 9.435l.926.843.927-.843-.927-.843zm5.003 1.577a1.538 1.4 0 00-1.51.675l-.39-.565h-.806v4.166h1.26V13.22a1.297 1.18 0 01.473-1.062 1.13 1.029 0 01.583-.127c.723 0 .927.38.927.995v2.26h1.223V12.69c0-1.164-.63-1.678-1.77-1.678z" fill="#481335"/></g></svg>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/luminor.svg:1]: We need to add `viewBox="0 0 38 24"`

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 38 24" width="38" height="24" aria-labelledby="pi-luminor"><rect width="100%" height="100%" fill="none"/><g class="currentLayer"><path opacity=".07" d="M35 0H3C1.3 0 0 1.3 0 3v18c0 1.7 1.4 3 3 3h32c1.7 0 3-1.3 3-3V3c0-1.7-1.4-3-3-3z"/><path fill="#fff" d="M35 1c1.1 0 2 .9 2 2v18c0 1.1-.9 2-2 2H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h32"/><path fill="#fff" d="M35 1c1.1 0 2 .9 2 2v18c0 1.1-.9 2-2 2H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h32" fill-opacity=".1"/><path d="M18.222 11.004a1.593 1.45 0 00-1.501.632v.05a1.612 1.467 0 00-1.51-.64 1.584 1.442 0 00-1.492.632v.05l-.389-.573h-.806v4.217h1.177v-2.15c0-.844.306-1.19.926-1.19s.843.32.843.987v2.269h1.288v-2.092c0-.793.296-1.155.926-1.155.371 0 .871.101.871.986v2.26h1.26V12.69c0-1.164-.5-1.686-1.593-1.686zm-7.82 2.184a1.316 1.198 0 01-.472 1.062 1.26 1.147 0 01-.583.118c-.732 0-.927-.38-.927-.995v-2.251H7.18v2.589c0 1.163.639 1.686 1.77 1.686a1.52 1.383 0 001.51-.683l.388.574h.806v-4.166h-1.25zm19.456-2.184a2.52 2.294 0 00-1.853.649 2.437 2.218 0 00-.658 1.552 2.4 2.184 0 00.658 1.543 2.52 2.294 0 001.853.65 2.353 2.142 0 002.501-2.193 2.353 2.142 0 00-2.501-2.201zm0 3.423a1.25 1.138 0 01-1.232-1.222 1.25 1.138 0 112.492 0 1.26 1.147 0 01-1.242 1.222zM3.566 9.435h-1.26v5.853h4.437v-1.19H3.566zm16.888 5.853h1.307V11.07h-1.307zm13.628-3.601l-.38-.574h-.824v4.217h1.26v-1.62a1.529 1.391 0 01.472-1.172 1.76 1.602 0 011.084-.362v-1.172c-1.352 0-1.621.683-1.621.683zM20.186 9.435l.926.843.927-.843-.927-.843zm5.003 1.577a1.538 1.4 0 00-1.51.675l-.39-.565h-.806v4.166h1.26V13.22a1.297 1.18 0 01.473-1.062 1.13 1.029 0 01.583-.127c.723 0 .927.38.927.995v2.26h1.223V12.69c0-1.164-.63-1.678-1.77-1.678z" fill="#481335"/></g></svg>
```

**Comment by martabelfm** [comment]
> One last thing and the rest looks good.

Anything else?

**Comment by tauthomas01** [comment]
> Anything else?

It's missing `<title>` attribute.

See example here: https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md#markup

**Comment by martabelfm** [comment]
> > Anything else?
> 
> It's missing `<title>` attribute.
> 
> See example here: https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md#markup

This PR has been a disaster. Found out that at first I used svgo command correctly but later when doing new changes the SVGO removed some of the items that were required as then I used it by memory.

File: app/assets/images/payment_icons/luminor.svg [hunk 4]
```diff
@@ -0,0 +1 @@
...
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/luminor.svg:1]: We can remove the `class`.

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 38 24" role="img" width="38" height="24" aria-labelledby="pi-luminor"><title id="pi-luminor">Luminor</title><rect width="100%" height="100%" fill="none"/><g><path opacity=".07" d="M35 0H3C1.3 0 0 1.3 0 3v18c0 1.7 1.4 3 3 3h32c1.7 0 3-1.3 3-3V3c0-1.7-1.4-3-3-3z"/><path fill="#fff" d="M35 1c1.1 0 2 .9 2 2v18c0 1.1-.9 2-2 2H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h32"/><path fill="#fff" d="M35 1c1.1 0 2 .9 2 2v18c0 1.1-.9 2-2 2H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h32" fill-opacity=".1"/><path d="M18.222 11.004a1.593 1.45 0 00-1.501.632v.05a1.612 1.467 0 00-1.51-.64 1.584 1.442 0 00-1.492.632v.05l-.389-.573h-.806v4.217h1.177v-2.15c0-.844.306-1.19.926-1.19s.843.32.843.987v2.269h1.288v-2.092c0-.793.296-1.155.926-1.155.371 0 .871.101.871.986v2.26h1.26V12.69c0-1.164-.5-1.686-1.593-1.686zm-7.82 2.184a1.316 1.198 0 01-.472 1.062 1.26 1.147 0 01-.583.118c-.732 0-.927-.38-.927-.995v-2.251H7.18v2.589c0 1.163.639 1.686 1.77 1.686a1.52 1.383 0 001.51-.683l.388.574h.806v-4.166h-1.25zm19.456-2.184a2.52 2.294 0 00-1.853.649 2.437 2.218 0 00-.658 1.552 2.4 2.184 0 00.658 1.543 2.52 2.294 0 001.853.65 2.353 2.142 0 002.501-2.193 2.353 2.142 0 00-2.501-2.201zm0 3.423a1.25 1.138 0 01-1.232-1.222 1.25 1.138 0 112.492 0 1.26 1.147 0 01-1.242 1.222zM3.566 9.435h-1.26v5.853h4.437v-1.19H3.566zm16.888 5.853h1.307V11.07h-1.307zm13.628-3.601l-.38-.574h-.824v4.217h1.26v-1.62a1.529 1.391 0 01.472-1.172 1.76 1.602 0 011.084-.362v-1.172c-1.352 0-1.621.683-1.621.683zM20.186 9.435l.926.843.927-.843-.927-.843zm5.003 1.577a1.538 1.4 0 00-1.51.675l-.39-.565h-.806v4.166h1.26V13.22a1.297 1.18 0 01.473-1.062 1.13 1.029 0 01.583-.127c.723 0 .927.38.927.995v2.26h1.223V12.69c0-1.164-.63-1.678-1.77-1.678z" fill="#481335"/></g></svg>
```

**Comment by martabelfm** [comment]
> Almost there, just 1 thing and it's good on me.

Ok done.

**Comment by tauthomas01** [comment]
@martabelfm Can you rebase off master and merge the conflicts?

The rest looks good.

**Comment by martabelfm** [comment]
> @martabelfm Can you rebase off master and merge the conflicts?
> 
> The rest looks good.

Done

**Pull request merged**